introduction.md

```diff
@@ -4,3 +4,7 @@ You’ll also learn about Dojo’s AMD-based module architecture, discover how t
 to your Web site or application, and find out how to get help when things go wrong.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?